src/app/(admin)/admin/settings/rooms/[id]/DeleteRoomButton.tsx

```diff
@@ -0,0 +1,50 @@
+import { api } from "~/trpc/react";
+import AdminButton from "~/app/_components/admin/Button";;
+import ConfirmationModal from "../../../../../_components/admin/ConfirmationModal";
```

Comment:
We can use an alias here to make the import a bit nicer. In the tsconfig I've set "~" as an alias for the src folder.

```suggestion
import ConfirmationModal from "~/app/_components/admin/ConfirmationModal";
```